src/sentry/integrations/jira_server/client.py

```diff
@@ -11,6 +13,10 @@
 from sentry.utils.http import absolute_uri
 
 
+def md5(*bits):
```

Comment:
Is this method ever used? I can see from your work in the jira cloud client that the jira server client share functionality. I notice this exact method in the jira cloud client. That's why I ask.

Reply:
Whoops. This is a vestige of an earlier change.